image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml.htpl

```diff
@@ -180,8 +180,8 @@ spec:
         {{- end }}
         {{- if ._rox.central.notifierSecretsEncryption }}
         {{- if ._rox.central.notifierSecretsEncryption.enabled }}
-        - name: central-encryption-key
-          mountPath: /run/secrets/stackrox.io/central-encryption-key
+        - name: central-encryption-key-chain
```

Comment:
this change is not backwards compatible, but the feature is not documented so I think it's acceptable.